src/arch-riscv64.h

```diff
@@ -0,0 +1,16 @@
+#ifndef _ARCH_RISCV64_H
```

Comment:
Do we want some sort of copyright header here?  If nothing else perhaps the standard LGPL boilerplate?